clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1636,6 +1642,58 @@ private void processConsumerActions(Store store) throws InterruptedException {
     }
   }
 
+  protected void processConsumerActionOrEnqueue(ConsumerAction action) {
+    if (action == null) {
+      return;
+    }
+    if (serverConfig.isProcessConsumerActionWithoutEnqueueEnabled()) {
```

Comment:
do you mean `if ( ! serverConfig.isProcessConsumerActionWithoutEnqueueEnabled())`?